Unity-Technologies/com.unity.multiplayer.samples.coop
# feat: additive scene loading

### Description (*)

This PR adds additive scene loading to BossRoom. The BossRoom scene is now split into three scenes: DungeonEntrance, DungeonTransitionArea and DungeonBossRoom. The Dungeon and StaticNetworkObjects prefabs have been split into three and moved to their respective scene. These scenes are loaded when at least one player character is inside a specific collider related to those scenes, and are unloaded when all players leave the collider.

When a scene is unloaded, all the static objects get despawned, which includes imps, pots, and spawners. When the scene is reloaded again, these objects are respawned.

The lighting was also re-baked for the BossRoom.unity scene with the three new scenes loaded. 

### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-1903](https://jira.unity3d.com/browse/MTT-1903)
### Manual testing scenarios

1. Start a game with a host and a client
2. Move through the level
3. See the scenes load and unload as you move
### Questions or comments

### Contribution checklist
